gbpservice/nfp/configurator/agents/loadbalancer_v2.py

```diff
@@ -0,0 +1,978 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import os
+
+from oslo_log import log as logging
+
+from gbpservice.nfp.common import exceptions
+from gbpservice.nfp.configurator.agents import agent_base
+from gbpservice.nfp.configurator.lib import data_filter
+from gbpservice.nfp.configurator.lib import lbv2_constants as lb_constants
+from gbpservice.nfp.configurator.lib import utils
+from gbpservice.nfp.core import event as nfp_event
+from gbpservice.nfp.core import poll as nfp_poll
+
+LOG = logging.getLogger(__name__)
+
+""" Implements LBaaS response path to Neutron plugin.
+Methods of this class are invoked by the LBaasEventHandler class and also
```

Comment:
Please change LBaasEventHandler to LBaaSV2EventHandler. Please change it wherever needed.